### DISCUSSION TITLE
Enable linux-s390x on conda-forge CDN

### DISCUSSION THREAD

**Discussion by isuruf** [conda/infra#4] [General]
https://conda-web.anaconda.org and https://pkgs.anaconda.com both support linux-s390x, but https://conda-static.anaconda.org doesn't.

Can you please enable linux-s390x on https://conda-static.anaconda.org?

**Comment by jezdez** [comment]
Hey @isuruf, thanks for opening the issue, I've moved it here to the GitHub discussions to make sure we can discuss first what is needed to get this enabled, and find out why it wasn't enabled before. And then open tickets that come out of this discussion. Those tickets would then be synced to Anaconda's internal ticket system.

@bandwidthninja @dbast Do you happen to know what's needed for this to be enabled?

**Reply by dbast** [reply]
Answered Jannis in private chat.

**Comment by chenghlee** [comment]
As discussed in yesterday's core meeting, CDN support for linux-s390x should now be enabled.
